packages/k4simdelphes/package.py

```diff
@@ -30,15 +32,17 @@ class K4simdelphes(CMakePackage, Ilcsoftpackage):
     variant('delphes_hepmc', default=True, description='Build standalone executable with Hepmc input.')
     variant('delphes_pythia_evtgen', default=True, description='Build standalone executable with Pythia+EvtGen input')
 
-    depends_on('edm4hep')
-    depends_on('delphes@3.4.3pre10:')
+    depends_on('edm4hep', type=('build', 'link', 'run'))
+    depends_on('podio', type=('build', 'link', 'run'))
+    depends_on('delphes@3.4.3pre10:', type=('build', 'link', 'run'))
```

Comment:
```suggestion
    depends_on('delphes@3.5:', type=('build', 'link', 'run'))
```
Should I try to make a more fine-grained thing about which k4simdelphes version needs which delphes version? Or are we satisfied with the fact that probably only the latest will build in any case?

Reply:
Given that there are several delphes versions which work but may give wrong results, I think it'll be good to be explicit here. But there's also no need to keep all of the pre-releases in the recipes - I'll take a look at it in a future PR.